src/app/models/Message.ts

```diff
@@ -0,0 +1,9 @@
+export class Message {
```

Comment:
Juz wczesniej napisalem ale powtorze - w TS nie stosuje sie takich konstrukcji. Chyba ze ma to na celu dodanie do Message jakiejs dodatkowej logikiw klasie. Zdecydowanie lepsza opcja z interfejsem (jak przy produktach)

Reply:
Poprawione